src/actors/FilterBarActor.js

```diff
@@ -85,21 +85,29 @@ export class FilterBarActor {
     });
   }
 
-  exportResults() {
+  exportResults(option) {
     if (this.exportPageLimitExceeded()) {
       alert(this.filterBarStore.getExportPageLimitExceededMessage());
     } else if (this.filterBarStore.persistent) {
-      URLHelper.redirectUrl(this.exportUrl());
+      URLHelper.redirectUrl(this.exportUrl(option));
     }
   }
 
+  exportUrl(option) {
+    const exportPartial = option === 'current' ? 'true' : 'false';
+    let url = URLHelper.updateUrlSearch(
+      this.filterBarStore.getExportResultsUrl(),
+      "q",
+      this.filterBarStore.getQuery()
+    ).toString();
+    url += `&export_partial=${exportPartial}`;
```

Comment:
you can call it again
`url = URLHelper.updateUrlSearch(url, "export_partial", exportPartial).toString()`

Reply:
@hanpeic https://github.com/rdytech/react-filterbar/pull/138/commits/d6444960a04ceb4eed95d604253b2a12a297fa8d